charts/osm/values.schema.json

```diff
@@ -265,20 +267,29 @@
                                 "IfNotPresent"
                             ]
                         },
-                        "defaultOutput": {
-                            "$id": "#/properties/OpenServiceMesh/properties/fluentBit/properties/defaultOutput",
+                        "outputName": {
+                            "$id": "#/properties/OpenServiceMesh/properties/fluentBit/properties/outputName",
                             "type": "string",
-                            "title": "The default output schema for Fluent Bit",
-                            "description": "The default log forwarding destination for Fluent Bit",
+                            "title": "The output schema for Fluent Bit",
```

Comment:
How come this says output schema instead of output name?

Reply:
Updated it